src/string/wcsnlen.c

```diff
@@ -2,7 +2,11 @@
 
 size_t wcsnlen(const wchar_t *s : itype(_Nt_array_ptr<const wchar_t>), size_t n)
 _Checked{
+
+// n is used in the declared bounds of d. But n is modified in the loop and hence the declared bounds of d no
```

Comment:
Please indent the comment with the code in this function. Also make sure each line of the comment is no longer than 80 chars.